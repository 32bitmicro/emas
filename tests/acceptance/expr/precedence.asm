.word 2 * 2 + 2
.word 2 + 2 * 2
.word (2 + 2) * 2
.word 1<<1 | 1<<2
.word ~1 | 1
.word 1\14 + 1\13
