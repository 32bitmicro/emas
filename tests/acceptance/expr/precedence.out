@ 0x0000 : 0x0006  /  000 000 0 000 000 110  /  6
@ 0x0001 : 0x0006  /  000 000 0 000 000 110  /  6
@ 0x0002 : 0x0008  /  000 000 0 000 001 000  /  8
@ 0x0003 : 0x0006  /  000 000 0 000 000 110  /  6
@ 0x0004 : 0xffff  /  111 111 1 111 111 111  /  -1
@ 0x0005 : 0x0006  /  000 000 0 000 000 110  /  6
